tests/core/config/models/bdk_authentication_config_test.py

```diff
@@ -10,18 +10,16 @@ def test_rsa_configuration():
     # Set up a non_empty content
     authentication_config.private_key.content = "not_empty"
     assert authentication_config.is_rsa_authentication_configured() is True
-    assert authentication_config.is_rsa_configuration_valid() is False
```

Comment:
I removed this statement as setting the content overrides the path, thus config is still valid